.github/workflows/maven.yml

```diff
@@ -1,6 +1,6 @@
 name: Java CI
 
-on: [push, pull_request]
+on: [push]
```

Comment:
Hi @sarasara100 :

Please pull the latest code before modifying,
Your changes here will prevent our CI from working.

FYL. @Functor10